<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndPage</class>
 <widget class="QWizardPage" name="EndPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Conclusion</string>
  </property>
  <property name="subTitle">
   <string>Please check your defined settings.</string>
  </property>
  <widget class="QGroupBox" name="gbEndPage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>7</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Overview:</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRemoteIP">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>21</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRemotePort">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>22</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Protocol:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRemoteProtocol">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>40</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>58</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CA:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPathToCA">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>56</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">:enabled {border: 1px solid black;background-color: rgb(195, 195, 195);color: rgb(0, 0, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);color: rgb(0, 0, 255);}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Certificate:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPathToCert">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>78</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">:enabled {border: 1px solid black;background-color: rgb(195, 195, 195);color: rgb(0, 0, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);color: rgb(0, 0, 255);}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>102</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPathToKey">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">:enabled {border: 1px solid black;background-color: rgb(195, 195, 195);color: rgb(0, 0, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);color: rgb(0, 0, 255);}
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>122</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Certificate type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCerttype">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>122</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>Text</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>138</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Advanced:</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>39</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>MSSFIX:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblMssfix">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>38</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Route method:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblRouteMethod">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>38</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>96</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mute:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Verbosity:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>77</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Route delay:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblRouteDelay">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>78</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblMute">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblVerbose">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>61</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>58</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Auth user/pass:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Comp-LZO:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>61</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Float:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>No bind:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblUserPass">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblNobind">
     <property name="geometry">
      <rect>
       <x>188</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFloat">
     <property name="geometry">
      <rect>
       <x>187</x>
       <y>61</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCompLzo">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>81</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Windir:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblWindir">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>default</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblInfo">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>265</y>
      <width>370</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MTU [1500], dev tun, persist key and persist tun will be added automatically.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
